internal/pkg/cli/app_show.go

```diff
@@ -121,6 +129,23 @@ func (o *showAppOpts) description() (*describe.App, error) {
 	if err != nil {
 		return nil, fmt.Errorf("list jobs in application %s: %w", o.name, err)
 	}
+	//1st approach takes 2 seconds to display the table
+	for _, env := range envs {
+		deployedJobs, err := o.deployStore.ListDeployedJobs(o.name, env.Name)
+		if err != nil {
+			return nil, fmt.Errorf("list deployed jobs %s: %w", o.name, err)
```

Comment:
nit: i think it would be more helpful to show them what env are we checking, because the app name is very obvious...
```suggestion
			return nil, fmt.Errorf("list jobs deployed to %s: %w", env.Name, err)
```